flask/app/families.py

```diff
@@ -1,4 +1,5 @@
 from dataclasses import asdict
+import inspect
```

Comment:
unused

Reply:
resolved in [ae207ab](https://github.com/ccmbioinfo/ST2020/pull/195/commits/ae207abacd76978cf9409db8654f61ca59f5a1c5)